buildlib/tools/test_wire_compat.sh

```diff
@@ -8,10 +8,11 @@
 exe_name=$1
 client_opt=$2
 common_opt=$3
-port=$((10000 + 1000 * ${AZP_AGENT_ID}))
+port=$((10000 + 1000 * ${AZP_AGENT_ID} + 100 * ${WIRE_COMPAT_STAGE_ID}))
```

Comment:
I know I'm late with my comment, since it's porting existing change.
But is it reliable way to get free port, and avoid collisions?
What if those ports are taken by other services running on the machine?
I remember implementing something similar in the past, and the approach was:
- Check `LOCAL_PORT_MIN LOCAL_PORT_MAX` from `/proc/sys/net/ipv4/ip_local_port_range`
- Check if port is used with `netstat -an | grep "LISTEN "`
- Synchronize between multiple processes by using `flock` on common file

Reply:
i think the proposed solution is good enough for now. We can change it if some issue arises

Reply:
we can implement a better solution on master branch